build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
+        classpath 'com.android.tools.build:gradle:3.3.3'
```

Comment:
Android 11 requires upgrading gradle build tools to this minor version (or 4.0.1+). We can't upgrade to the latest 3.x core version without doing this, but are we ok with this as a non-breaking change? I think merchants will have to upgrade their build tools to the latest minor version also

Reply:
I think we should call it out in the CHANGELOG or elsewhere if this BT DropIn update requires a tooling update on their end.